restli-server/src/main/java/com/linkedin/restli/internal/server/methods/MethodBuildersRegistry.java

```diff
@@ -0,0 +1,23 @@
+package com.linkedin.restli.internal.server.methods;
+
+import com.linkedin.restli.common.ResourceMethod;
+import com.linkedin.restli.internal.server.methods.arguments.RestLiArgumentBuilder;
+import com.linkedin.restli.internal.server.response.RestLiResponseBuilder;
+
+public interface MethodBuildersRegistry {
```

Comment:
After some offline discussion, I've decided that the new preferred naming for this interface is `MethodAdapterProvider`. Please update all subclasses/tests/methods accordingly.

Sorry for the overhead, just trying to get the naming right 😃

Reply:
Sounds good. Updated in dce626d